prow/getting_started_deploy.md

```diff
@@ -179,17 +179,19 @@ to start receiving GitHub events!
 Configure github to send your prow instance `application/json` webhooks
 for specific repos and/or whole orgs.
 
-You can do this with the `add-hook` utility:
+You can do this with the `update-hook` utility:
 
 ```sh
 # Note /path/to/hook/secret and /path/to/oauth/secret from earlier secrets step
 # Note the an.ip.addr.ess from previous ingres step
 
 # Ideally use https://bazel.build, alternatively try:
-#   go get -u k8s.io/test-infra/experiment/add-hook && add-hook
-bazel run //experiment/add-hook -- \
+#   go get -u k8s.io/test-infra/experiment/update-hook && update-hook
+bazel run //experiment/update-hook -- \
   --hmac-path=/path/to/hook/secret \
   --github-token-path=/path/to/oauth/secret \
+  --github-endpoint=http://ghproxy \  # Replace this as needed to point to your ghProxy instance.
+  --github-endpoint=https://api.github.com \
```

Comment:
During setup we wouldn't expect someone to have ghproxy deployed yet or even know what it is. Since this would likely be run locally the ghproxy instance wouldn't be reachable in this way either.  I'm not sure I'd ever expect ghproxy to be used with the `update-hook` utility. It may make more sense for the `hmac` controller though assuming GH serves ETags for the webhook API.

Reply:
I have removed these two lines, and set `AllowDirectAccess` to be true for the github client used here, so it won't print out warning messages.